packages/api/pw-test.config.cjs

```diff
@@ -11,54 +11,34 @@ dotenv.config({
 })
 
 const cli = path.join(__dirname, 'scripts/cli.js')
-
 /** @type {import('esbuild').Plugin} */
 const nodeBuiltinsPlugin = {
   name: 'node builtins',
   setup(build) {
     build.onResolve({ filter: /^stream$/ }, () => {
       return { path: require.resolve('readable-stream') }
     })
-
-    build.onResolve({ filter: /^cross-fetch$/ }, () => {
-      return { path: path.resolve(__dirname, 'scripts/fetch.js') }
```

Comment:
@redaphid do we not need this `cross-fetch` thing any more?

Reply:
Nope! later playwrights use the native fetch when possible. Just a side effect of going through the code.

Reply:
this is not related to playwright, its related to [postgrest-js](https://github.com/supabase/postgrest-js) and cross-fetch is still in there (https://github.com/supabase/postgrest-js/blob/master/src/lib/types.ts#L1) even if the code may use web fetch so this is still use full because it remove cross-fetch from the bundle.

Reply:
let's keep it to remove cross-fetch from bundle. And probably nice to add a comment on why that is needed aka postgrest-js to remove from the bundle